fontforge/autowidth.c

```diff
@@ -1086,7 +1086,7 @@ int AW_ReadKernPairFile(char *fn,WidthInfo *wi) {
     unichar_t buffer[300];
     struct kernsets ks;
 
-    filename = utf82def_copy(fn);
+    filename = utf82fsys_copy(fn);
```

Comment:
As mentioned already before - **all** places involving filenames **must** be UTF-8 encoded on Windows.